README.md

```diff
@@ -27,7 +27,7 @@ Once setup, create a [fork](https://docs.github.com/en/pull-requests/collaborati
 git clone git@github.com:albert-einstein/aframev2.git
 ```
 
-Set the `AFRAME_REPO` environment variable in your `~/.bash_profile` to point to the location where you cloned the repository.
+Set the `AFRAME_REPO` environment variable in your `~/.bash_profile` to point to the location where you cloned the repository. 

```

Comment:
Had a couple students confused about what this meant, just from not being familiar with the shell. Could you add the explicit command to add the variable to the file, and same with `AFRAME_CONTAINER_ROOT` below?

Reply:
And also an instruction to `source ~/.bash_profile`